lua/nvim-navic/init.lua

```diff
@@ -472,7 +472,7 @@ function M.get_location(opts)
 		local_config = config
 	end
 
-	local data = M.get_data()
+	local data = M.get_data(opts.bufnr)
```

Comment:
Instead of having bufnr inside opts, I would like to have it as a seperate argument to get_location.

get_location(opts, bufnr)

This way, opts is only reponsible for overriding all the config options.

Reply:
Makes sense. 👍 I made the change.